packages/react-urql/examples/2-using-subscriptions/package.json

```diff
@@ -35,6 +35,6 @@
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
     "subscriptions-transport-ws": "^0.9.16",
-    "urql": "link:../../"
+    "urql": "latest"
```

Comment:
Another note, if none of our examples use `link:` anymore, their Webpack configs can be simplified and we won't have to build the entire repo anymore;
I think the important thing is that examples just work and secondly that they reflect the repo state :)